src/miral/symbols.map

```diff
@@ -475,7 +475,9 @@ global:
   extern "C++" {
     miral::Output::id*;
     miral::WindowInfo::attached_edges*;
+    miral::WindowInfo::clip_area*;
```

Comment:
These should be added to a new 2.8 stanza. (This is not your fault, we have failed to update src/miral/regenerate-miral-symbols-map.py since the last release - see #995.)

Once that is fixed they are also needed in debian/libmiral3.symbols

Reply:
You have to commit the debian symbols to git, but the changes should be done either automatically or by `make regenerate-miral-symbols-map`

Reply:
I rebased and regenerated the symbols to get them into 2.8. But I can't figure out how to regenerate the symbols in `debian/libmiral3.symbols`. It doesn't do it automatically and `make regenerate-miral-symbols-map` only regenerates `src/miral/symbols.map`.

Reply:
I had a poke around, the regeneration only happens if you're on a debian based distro and the program `dpkg-gensymbols` is installed. (See `src/miral/CMakeLists.txt` for the gory details.)

Reply:
Hmm, I'm on Ubuntu and do have `dpkg-gensymbols` installed. Manually running `dpkg-gensymbols -plibmiral3` did include the new symbols either for some reason. However your tip about the cmake file led me to the python script `src/miral/check-and-update-debian-symbols.py` and manually running `python3 src/miral/check-and-update-debian-symbols.py build/lib miral 2.8.0 3` did generate the symbols.

If it's due to some bad setup on my computer then I guess it doesn't matter as the symbols are there now but it might be worth investigating if there's some buildchain problem (maybe with the update to 2.8?). I don't know anything about about gensymbols so it's very possible that I have screwed something up though.

Reply:
IIRC you also have to be using GCC. If you are then idk